tests/scripts/perf-slices.json

```diff
@@ -0,0 +1,48 @@
+{
+    "slices" : [
+        {
+            "id" : "1",
```

Comment:
What is the id field for?

Reply:
I thought I would use them, but ultimately have not